import_bsp/BspGeneric.py

```diff
@@ -53,7 +74,7 @@ def pack_lightmaps(bsp, import_settings):
                     pixels.append(working_pixels[id * 4 + 1])
                     pixels.append(working_pixels[id * 4 + 2])
                     pixels.append(working_pixels[id * 4 + 3])
-                    
+
             lightmaps.append(pixels)
```

Comment:
Pls add a check if the lightmap exists and set the lightmap directly instead of appending the pixel list.
Also check if the rest of the function will work with Nones in the "lighmaps" list.

Reply:
Those are feature requests and do not belong to this PR.